e2e/cypress/integration/account_settings/general/main_menu_stays_open_spec.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+import users from '../../../fixtures/users.json';
+
+let townsquareChannelId;
+
+describe('Account Settings > General', () => {
+    before(() => {
+        // # Login and go to /
+        cy.apiLogin('sysadmin');
+        cy.visit('/');
+
+        // # Store channel id for further use
+        cy.getCurrentChannelId().then((id) => {
+            townsquareChannelId = id;
+        });
+    });
+
+    it('AS15009 - Main Menu stays open', () => {
+        // # Click the hamburger button
+        cy.get('#headerInfo').find('button').click({force: true});
+
+        // # Wait for the menu to open
+        cy.wait(TIMEOUTS.TINY);
```

Comment:
nit: can be removed since getting `#sidebarDropdownMenu` has an auto-retry until the element exist.
```suggestion
     
```